pom.xml

```diff
@@ -705,6 +705,12 @@
 
             <dependency>
                 <groupId>org.wildfly.core</groupId>
+                <artifactId>wildfly-core-testsuite-shared</artifactId>
+                <version>${project.version}</version>
+            </dependency>
+
```

Comment:
OT: this is just my personal opinion and isn't any sort of policy, but I quite dislike org.wildfly.core:wildfly-core-testsuite-shared being used outside of WildFly Core. I don't think people working on a testsuite utility module will put the kind of care into it that's appropriate for public API used by other projects.